parsers/keyval/timeformat.go

```diff
@@ -0,0 +1,66 @@
+package keyval
```

Comment:
Would be nice to not duplicate this between `parsers/htjson` and `parsers/keyval`, but that can happen later :)